pygmt/datasets/load_remote_dataset.py

```diff
@@ -210,9 +223,8 @@ class GMTRemoteDataset(NamedTuple):
         },
     ),
     "earth_wdmam": GMTRemoteDataset(
-        title="WDMAM magnetic anomaly",
-        name="wdmam",
-        long_name="World Digital Magnetic Anomaly Map",
+        name="earth_wdmam",
+        long_name="World Digital Magnetic Anomaly Map (WDMAM)",
```

Comment:
https://www.generic-mapping-tools.org/remote-datasets/earth-wdmam.html
```suggestion
        long_name="WDMAM world digital magnetic anomaly map",
```